ProxyHandler.class.php

```diff
@@ -73,6 +73,10 @@ function __construct($options)
         if (is_string($options)) {
             $options = array('proxyUri' => $options);
         }
+        if (isset($options['forceBuffered']))
+            $this->_buffered = true;
+        if (isset($options['forceContentType']))
+            $this->_contentType = $options['forceContentType'];
```

Comment:
If this type is in _bufferedContentTypes, should set buffered to true at this point.